### ISSUE TITLE
No minor bump on develop when a new release branch is created

### ISSUE THREAD

**Issue by sl-nigelalmada** [GitTools/GitVersion#1990]
I'm using this Docker Container Image in order to generate a gitversion for each push on my repo : https://hub.docker.com/layers/gittools/gitversion/latest-linux/images/sha256-93aff651032f58cf96655ea1e7d1fbe711439e6152037352fc531d74b8c7209d (lastest-linux should be 5.1.3 right now) with the default config

The issue is that the minor version number is not incremented when a release branch (release/x.y.0) is created

I know that this behaviour is possible from https://github.com/GitTools/GitVersion/pull/832 

And this image summarize exactly what I want to reproduce : ![](https://cloud.githubusercontent.com/assets/453152/16898779/3e9bb364-4c1b-11e6-8138-a8c7bf86dfdb.png)

Basically I have the "deleted" behaviour (minus the "unstable", I've "alpha" instead) while I want the "added" one.

I've tried to override the config with the GitVersion.yml file , but I can't found how to do it.

Thanks

**Comment by asbjornu** [comment]
Are you able to submit a PR with a `RepositoryFixture` that reproduces your problem?
